source/_integrations/jewish_calendar.markdown

```diff
@@ -70,25 +71,25 @@ havdalah_minutes_after_sunset:
 Time sensor states are represented as ISO8601 formatted *UTC time*.
 For easier use in automations, all time sensors have a `timestamp` attribute, which returns the UNIX timestamp.
 
-- first_light: First light of dawn (Alot Hashachar - עלות השחר).
-- talit: Earliest time for tallit and tefillin
-- gra_end_shma: Last time for reading of the Shma according to the Gr"a.
-- mga_end_shma: Last time for reading of the Shma according to the MG"A.
-- gra_end_tefilla: Last time for full shacharit according to the Gr"a.
-- mga_end_tefilla: Last time for full shacharit according to the MG"A.
-- big_mincha: Earliest time for Mincha (Mincha Gedola)
-- little_mincha: Preferable earliest time for Mincha (Mincha Ketana)
-- plag_mincha: Time of the Plag Hamincha.
-- sunset: Sunset (Shkiya)
-- first_stars: Time at which the first stars are visible (Tseit Hakochavim - צאת הכוכבים).
-- upcoming_shabbat_candle_lighting: The time of candle lighting for either the current Shabbat (if it is currently Shabbat) or the immediately upcoming Shabbat.
-- upcoming_shabbat_havdalah: The time of havdalah for either the current Shabbat (if it is currently Shabbat) or the immediately upcoming Shabbat. If it is currently a three-day holiday, this value *could* be None (i.e., if holiday is Sat./Sun./Mon. and it's Saturday, there will be no shabbat_havdalah value. See comments in hdate library for details.)
-- upcoming_candle_lighting: The time of candle lighting for either the current Shabbat OR Yom Tov, or the immediately upcoming Shabbat OR Yom Tov. If, for example, today is Sunday, and Rosh Hashana is Monday night through Wednesday night, this reports the candle lighting for Rosh Hashana on Monday night. This avoids a situation of triggering pre-candle-lighting automations while it is currently Yom Tov. To always get the Shabbat times, use the upcoming_shabbat_candle_lighting sensor.
-- upcoming_havdalah: The time of havdalah for either the current Shabbat OR Yom Tov, or the immediately upcoming Shabbat OR Yom Tov. If, for example, today is Sunday, and Rosh Hashana is Monday night through Wednesday night, this reports the havdalah for Rosh Hashana on Wednesday night. To always get the Shabbat times, use the upcoming_shabbat_havdalah sensor.
+- `alot_hashachar`: First light of dawn (Alot Hashachar - עלות השחר)
```

Comment:
Could you elaborate on this change or provide a description?

According to the code base, this sensor is `first_light`, not `alot_hashachar`.

![image](https://user-images.githubusercontent.com/195327/92513161-286f4200-f210-11ea-9854-dec5574d7502.png)

Reply:
I se, you are right - but also:
![Screenshot_2020-09-09-05-52-49-267_io homeassistant companion android](https://user-images.githubusercontent.com/3170485/92552433-52a11e00-f269-11ea-897e-05b341235b29.jpg)

It might be due to languige settings in the integration, which make it use the name instead.
I consider this a bug as it is not hard coded.. one might get one sensor name and the other different one, and the docs do not reflect that.

I see two options:
To hard code the sensor name that the integration exposes so it will be in english or hebrew only.
To note in the docs that different settings exposes different sensor names.

(just tagging @tsvi as he is the maintainer)

Reply:
Entity names are customizable, can be inherited from older defaults, or renamed as a result of device renaming.

The documentation cannot incorporate customizations.

Reply:
So if I understood you correctly - I just need to use the sensor name as in the code base becouse this is the default one? will do.